src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplGetCaseByUprnTest.java

```diff
@@ -434,6 +440,43 @@ private void verifyNewCase(AddressIndexAddressCompositeDTO address, CaseDTO resu
         address.getCensusAddressType(), address.getCensusEstabType(), newAddress);
   }
 
+  private void verifyCachedCaseContent(
+      AddressIndexAddressCompositeDTO expectedAddress,
+      UUID expectedId,
+      CaseType expectedCaseType,
+      CachedCase actualCapturedCase) {
+    assertEquals(expectedId.toString(), actualCapturedCase.getId());
+    assertEquals(expectedAddress.getUprn(), actualCapturedCase.getUprn());
+    assertEquals(createFormattedAddress(expectedAddress), actualCapturedCase.getFormattedAddress());
+    assertEquals(expectedAddress.getAddressLine1(), actualCapturedCase.getAddressLine1());
+    assertEquals(expectedAddress.getAddressLine2(), actualCapturedCase.getAddressLine2());
+    assertEquals(expectedAddress.getAddressLine3(), actualCapturedCase.getAddressLine3());
+    assertEquals(expectedAddress.getTownName(), actualCapturedCase.getTownName());
+    assertEquals(expectedAddress.getPostcode(), actualCapturedCase.getPostcode());
+    assertEquals(expectedAddress.getCensusAddressType(), actualCapturedCase.getAddressType());
+    assertEquals(expectedCaseType, actualCapturedCase.getCaseType());
+    assertEquals(expectedAddress.getCensusEstabType(), actualCapturedCase.getEstabType());
+    assertEquals(expectedAddress.getCountryCode(), actualCapturedCase.getRegion());
+    assertEquals(expectedAddress.getOrganisationName(), actualCapturedCase.getCeOrgName());
+    assertEquals(0, actualCapturedCase.getCaseEvents().size());
+  }
+
+  private String createFormattedAddress(AddressIndexAddressCompositeDTO expectedAddress) {
```

Comment:
It's my understanding from talking to @philwhiles  what the formattedAddress is a mistake in the CachedCase, and will normally be null.  I don't think it is used anywhere, and you probably don't want to write this test code for it.

Reply:
It might be a mistake but it is currently being used. 
I've just run CC in the debugger and forced the creation of a new cached case, and that is being created with a formatted address.
Maybe this ought to remain until/if formatted address is deleted?

Reply:
I've deleted the code which builds the formatted address. 
It now compares the actual formatted address with the expected.formattedAddress, which was there all along !!
If formattedAddress gets deleted in the future then this single line of test code will go too.